components/admin_console/admin_sidebar/admin_sidebar.jsx

```diff
@@ -205,11 +205,16 @@ class AdminSidebar extends React.PureComponent {
                         }
                     }
                     const subDefinitionKey = `${key}.${subKey}`;
+                    let tag = '';
+                    if (this.props.config.FeatureFlags.OpenIdConnect === 'on') {
```

Comment:
I'd rather have this as a function incase we need other features to have a tag. Maybe a tag can be passed in as a object with a value and condition or two different separate keys but this should be dynamic such that we can use it in other places if we need too in the future.
```
tag : {
    value: <FormattedMessage ../>
    shouldDisplay: config.FeatureFlag.OpenIdConnect === 'on'
}
```

Then here you can do 
```suggestion
                    if (item.tag?.shouldDisplay){
                       tag = item.tag.value
                   }
```

This way if any future sidebar menu items need a tag, it will be flexible and easy to add.